src/Components/AboutMeSection.tsx

```diff
@@ -8,113 +8,96 @@ export default function AboutMeSection() {
-              </p>
+            <div className="mt-5 prose prose-indigo text-md text-gray-500 mx-auto lg:max-w-none lg:row-start-1 lg:col-start-1">
               <p>
-                Bibendum eu nulla feugiat justo, elit adipiscing. Ut tristique
-                sit nisi lorem pulvinar. Urna, laoreet fusce nibh leo. Dictum et
-                et et sit. Faucibus sed non gravida lectus dignissim imperdiet
-                a.
+                I’m Maximilian Gruber (yes, like the bad guy in Die Hard). I’m a
+                NaN (😉) year old Junior Full Stack Web Developer graduated from
+                Ironhack’s 10-week immersive bootcamp in Paris, France. In a
+                past life I had a career in Hospitality, and am now keen to
+                start off my digital career using my new found skills in MERN
+                Stack.
               </p>
               <p>
-                Dictum magnis risus phasellus vitae quam morbi. Quis lorem lorem
-                arcu, metus, egestas netus cursus. In.
+                I am a tri-cultural person fluent in English, French and German
+                allowing me to insert myself easily in any work culture or
+                conversation, while also having a heck of a time deciding who
+                I'm backing in football competitions.
               </p>
-              <ul>
-                <li>Quis elit egestas venenatis mattis dignissim.</li>
-                <li>
-                  Cras cras lobortis vitae vivamus ultricies facilisis tempus.
-                </li>
-                <li>Orci in sit morbi dignissim metus diam arcu pretium.</li>
-              </ul>
               <p>
-                Rhoncus nisl, libero egestas diam fermentum dui. At quis
-                tincidunt vel ultricies. Vulputate aliquet velit faucibus
-                semper. Pellentesque in venenatis vestibulum consectetur nibh
-                id. In id ut tempus egestas. Enim sit aliquam nec, a. Morbi enim
-                fermentum lacus in. Viverra.
+                Given the right guidance, my goal is to become a successful Full
+                Stack Developer.
+                <br />
+                Technology leads, society follows. I want to be at the heart of
+                that change. I want to make a difference.
               </p>
-              <h3>How we helped</h3>
               <p>
-                Tincidunt integer commodo, cursus etiam aliquam neque, et.
-                Consectetur pretium in volutpat, diam. Montes, magna cursus
-                nulla feugiat dignissim id lobortis amet. Laoreet sem est
-                phasellus eu proin massa, lectus. Diam rutrum posuere donec
-                ultricies non morbi. Mi a platea auctor mi.
+                Thanks to my past experiences in management within the
+                Hospitality sector, I know what it is like to have to learn
+                things quickly, solve issues spontaneously yet correctly and,
```

Comment:
```suggestion
                things quickly, solve issues spontaneously and
```
there should not be commas after or before "and"